homeassistant/components/hive/__init__.py

```diff
@@ -69,27 +70,12 @@
 )
 
 
-class HiveSession:
-    """Initiate Hive Session Class."""
-
-    entity_lookup = {}
-    core = None
-    heating = None
-    hotwater = None
-    light = None
-    sensor = None
-    switch = None
-    weather = None
-    attributes = None
-    trv = None
-
-
-def setup(hass, config):
+async def async_setup(hass, config):
     """Set up the Hive Component."""
 
-    def heating_boost(service):
+    async def heating_boost(service):
```

Comment:
In a future PR, the entity services should be registered with our entity service helper.

https://developers.home-assistant.io/docs/dev_101_services#entity-services